package.json

```diff
@@ -50,7 +50,7 @@
     "eslint": "^3.19.0",
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
-    "jest": "^19.0.2",
+    "jest": "^20.0.3",
     "lint-staged": "^3.4.0",
     "nodemon": "^1.11.0",
```

Comment:
The issue is that the scripts (`serve:dist` :hocho: , `clean:dist` => `clean` etc) and `nodemon`  shouldn't be there anymore after #49 landed.  https://github.com/webpack-contrib/webpack-defaults/pull/49/files#diff-709d5c3e0ecd86cd23e75d1951715f43R39